src/App.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css?family=Josefin+Sans');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  font-family: 'Josefin Sans', cursive;
+  background-color: #fc5193;
+}
+
+.card-container.card {
+  height: 400px;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+a:hover {
+  text-decoration: none;
+  color: white;
+}
+
+.arrow-right {
+  float: right;
+}
+
+.icon {
+  color: white;
+  padding: 2px;
+  text-align: center;
+  font-size: 25px;
+  margin-bottom: 7px;
+}
+
+.year {
+  color: white;
+  padding: 2px;
+  text-align: center;
+  font-size: 25px;
+}
+
+.year:hover {
+  color: white;
+}
+
+.btn {
+  color: white !important;
+  border-color: #dc4782 !important;
+}
+
+.btn:hover {
+  background-color: #dc4782 !important;
+  border-color: #dc4782 !important;
+}
+
+.africa-image {
+  width: 100%;
+  height: 50%;
+}
+
+.color1 {
+  background-color: #dc4782 !important;
+}
+
+.country-details {
+  color: white;
+}
+
+.country-details:nth-child(odd) {
```

Comment:
- [ ] Kindly work on the styling, try as much as possible to match the project design especially on the home page. This is a mobile version of the app, let's pay attention to that as well.